pkg/agent/multicluster/mc_route_controller.go

```diff
@@ -437,9 +613,31 @@ func generatePeerConfigs(subnets []string, gatewayIP net.IP) (map[*net.IPNet]net
 }
 
 // getPeerGatewayIP will always return the first Gateway IP.
-func getPeerGatewayIP(spec mcv1alpha1.ClusterInfo) net.IP {
+func getPeerGatewayIP(spec mcv1alpha1.ClusterInfo, enableWireGuard bool) net.IP {
+	if enableWireGuard {
+		_, serviceCIDR, err := net.ParseCIDR(spec.ServiceCIDR)
+		if err != nil {
+			return nil
```

Comment:
Better to add an info or warning log before it.

Reply:
Add a warning log.